lib/analytics/page.rb

```diff
@@ -0,0 +1,72 @@
+module Analytics
+  class Page
```

Comment:
how expensive would it be to pull the data for all pages to create a global overview and page ranking? Something like
```
Page.all.map do |page|
  analytics = Analytics::Page.new(page.id)
  [analytics.total_actions, analytics.total_new_members, analytics.total_actions_over_time]
end
```
We could make some sweet charts and leaderboards with that info for a landing page dashboard, but it also means reading every effing key every time that someone loads the homepage. Is that a consideration? Is there a way we could track an overview stat at write time?